src/technologies/overview/tools-libraries/index.md

```diff
@@ -1,5 +1,61 @@
 ---
-title:
+title: Tools and libraries
 ---
 
-{% include content-not-available.md %}
\ No newline at end of file
+To work with the utilities provided by the Magento PWA project, you should be familiar with the tools and libraries described on this page.
+
+## Webpack
+
+[Webpack] is a configuration-driven module bundler for JavaScript applications.
```

Comment:
`webpack` bundles JavaScript, but also frequently bundles (or optimizes) CSS/HTML/images/etc. Might make more sense to call it an asset bundler, based on the tag line on the homepage.

![image](https://user-images.githubusercontent.com/5233399/38954491-618d0018-4317-11e8-8928-3a4d7654cde1.png)